jsondown.js

```diff
@@ -1,18 +1,66 @@
-var util = require('util');
-var os = require('os');
-var MemDOWN = require('memdown');
-var fs = (os.hostname() === 'runtime') ?
-  require('./runtime-fs') :
-  require('fs')
+var AbstractLevelDOWN = require("abstract-leveldown").AbstractLevelDOWN;
+var os = require("os");
+var util = require("util");
+var path = require("path");
+var mkdirp = require("mkdirp");
+var MemDOWN = require("memdown");
+var fs = os.hostname() === "runtime" ? require("./runtime-fs") : require("fs");
 
+function serializeRBTree(tree) {
+  var result = {};
+  tree.forEach((key, value) => {
+    result[key] = value;
+  });
+  return result;
+}
 
-var niceStringify = require('./nice-stringify');
+function serializeStore(store) {
+  var result = {};
+  Object.keys(store).forEach(location => {
```

Comment:
The tests are failing on Travis because it's running with Node 0.10.0. You should either update the `.travis.yml` file to a later Node version or remove the arrow functions (this one and the one on line 11)

Reply:
Oh geez, I wrote this a billion years ago. I'm fine with upgrading the version of Node that this package expects--I assume this would involve not only updating `.travis.yml`, but also adding something to `package.json` indicating the version of node that's needed?